tests/payroll/test_models.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+from django.db.utils import IntegrityError
+
+
+@pytest.mark.django_db(transaction=True)
```

Comment:
Since you're testing the creation of objects but don't need to access them, I _think_ you should be able to omit `transaction=True` here. [Here's a bit more](https://github.com/datamade/testing-guidelines/blob/master/framework-specific-patterns.md#transactional-context) on the behavior of transactional context in Django tests. 

(I'm unsure because check constraints are implemented at the database level, so it's possible you need `transaction=True` so the data is actually pushed to the database for the exception you're expecting to be raised... I'd suggest trying one without and seeing if your tests pass. Let me know what you find out!)

Reply:
it works without `transaction=True`!